File: solutions/alfi.compitest/source_gen/alfi/compitest/ActivityHello.cpp
#include "ActivityHello.h"
#include <alf/library/BasicInputOutput.hpp>
#include <string>

void ActivityHello::Hello()
{
  alf::library::BasicInputOutput::WriteLine(std::string("Hello!"));
  alf::library::BasicInputOutput::WriteLine(std::string("How are you?"));
  std::string input = alf::library::BasicInputOutput::ReadLine();

  if (input == std::string("good"))
  {
    alf::library::BasicInputOutput::WriteLine(std::string("Ok nice"));
  }
  else
  {
    alf::library::BasicInputOutput::WriteLine(std::string("Ok"));
  }
}

File: solutions/alfi.compitest/source_gen/alfi/compitest/ActivityHello.cs
/// <filename>
///     ActivityHello.cs
/// </filename>
using Alf.Library;

public class ActivityHello
{
    public static void Execute()
    {
        BasicInputOutput.WriteLine("Hello!");
        BasicInputOutput.WriteLine("How are you?");
        string input = BasicInputOutput.ReadLine();

        if (input == "good")
        {
            BasicInputOutput.WriteLine("Ok nice");
        }
        else
        {
            BasicInputOutput.WriteLine("Ok");
        }
    }

    public static void Main(string[] args)
    {
        ActivityHello.Execute();
    }
}
